### PR TITLE
Create associacao-brasileira-de-normas-tecnicas-ufs

### PR DESCRIPTION
**Pull request by JohnHed**
citation styles for federal college sergipe using the ABNT

### CODE REVIEW

**Comment by rmzelle** [comment]
@JohnHed, could you

* change the title to `<title>Associação Brasileira de Normas Técnicas - Universidade Federal de Sergipe (Portuguese - Brazil)</title>`?
* add `default-locale="pt-BR"` to the `<style>` element. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale
* change the file name to "associacao-brasileira-de-normas-tecnicas-ufs.csl"
* change the style ID and "self" link to use "http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufs"
* update the "documentation" link if your style is described online, or delete it otherwise
* Delete the `<category field="psychology"/>` line
* Add a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: Associacao-Brasileira-de-Normas-tecnicas-ufs.csl [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <title-short>ABTN-UFS</title-short>
     <id>http://www.zotero.org/styles/Associação-Brasileira-de-Normas-tecnicas-ufs</id>
     <link href="http://www.zotero.org/styles/original-style" rel="template"/>
-    <link href="http://www.zotero.org/styles/Associação-Brasileira-de-Normas-tecnicas-ufs" rel="self"/>
+    <link href="http://www.zotero.org/styles/Associacao-Brasileira-de-Normas-tecnicas-ufs" rel="self"/>
```
Comment (rmzelle) [inline Associacao-Brasileira-de-Normas-tecnicas-ufs.csl]: This should be all lowercase, i.e.:

`<link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufs" rel="self"/>`

**Comment by rmzelle** [comment]
Let us know if you'd like some help.

**Comment by JohnHed** [comment]
it was just a small detail that was forgetting, thanks for the tip

**Comment by rmzelle** [comment]
Thanks. Is this style format described online anywhere? We're still missing an updated "documentation" link and a "template" link.

* update the "documentation" link if your style is described online, or delete it otherwise. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link
* Add a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

**Comment by rmzelle** [comment]
Sorry, but you just linked the style to itself with your "template" link.

The point of the "template" link is to point to the CSL style you based your current style on, so that we more easily figure out what the history is behind the CSL code in the style. If you started from "apa.csl", it should be `<link href="http://www.zotero.org/styles/apa" rel="template"/>`.

**Comment by JohnHed** [comment]
Thanks , I made the change as explained

**Comment by rmzelle** [comment]
I just have a final question of whether there is an online description available of this citation style, so that you could add a "documentation" link.

**Comment by JohnHed** [comment]
has some styles of similar citations with my using this documentation  ->  <link href = " https://forums.zotero.org/discussion/22148/how-to-apply-disambiguation-in-citation-just-when- works- of- different- authors -are- in-the -same- year / ? Focus = 147094 # Comment_147094 "rel =" documentation " />
  
Can I use it?

**Comment by rmzelle** [comment]
Great, thanks. Sorry this took a while to get right.

**Pull request merged**